### ISSUE TITLE
[DevTools Bug]: CDN-based site not working

### ISSUE THREAD

**Issue by ForbiddenEra** [facebook/react#22834]
### Website or app

https://lcdev.shaped.ca

### Repro steps

Dev tools not working in FF or Chrome, says "This page doesn't appear to be using react".

React is included via CDN as shown on react website https://reactjs.org/docs/cdn-links.html:
``
	<script crossorigin="anonymous" src="https://unpkg.com/react@17/umd/react.development.js"></script>
``

Web Console on said page says:
``
Download the React DevTools for a better development experience: https://reactjs.org/link/react-devtools
``

### How often does this bug happen?

Every time

**Comment by kouraf** [comment]
I am trying to reproduce the bug local with this [code](https://github.com/kouraf/react-cdn) but it works fine !
![scrren](https://user-images.githubusercontent.com/22027736/144745307-e108e8de-c79e-43dd-ac11-0f959971dbac.png)

**Comment by lunaruan** [comment]
Hey! Thanks for reporting an issue! Unfortunately, I can't repro your issue and your repro steps is not enough information for us to help. Could you link to the source code for your website?

**Comment by ForbiddenEra** [comment]
Hi,

I doubt anyone would be able to reproduce it locally. I have a feeling it's likely due to how Freeform framework distributes content; as XHTML5 (Content-Type: application/xhtml+xml - not something you can really simulate without a web server) though it could be related to something else, I often find many apps from big companies completely forget (including stuff from Google, for example, performing ToUpper() on tagnames for no good reason, which is not valid HTML) that this exists and is a valid, and IMHO, better, if unfortunately uncommon, method of serving content.

If you visit the site linked and view it's source, you can see that I'm including react from a CDN. This alone should enable the React dev tools, but it's greyed out on both FF and Chrome. This doesn't change even if a React component is actually running on the page, I can create, run and interact with components fine, but the React tools are always greyed out on both browser types even when a component is functioning.

The best way to maybe reproduce on a local server would simply be to view source, copy and paste into an xhtml file, make sure your webserver is setup to serve the xhtml file as the aforementioned content type and view it that way; though, this is why I provided a link to a deployed site where the issue is visible, so it could be immediately seen upon visiting.

**Comment by ForbiddenEra** [comment]
Wow, lots of activity on this today!

So, yes, the initial page there doesn't have any React components, there was one I used when I logged in. The site was a 'code challenge' for a job that just emailed back today saying they chose someone else, so I've taken it offline anyway.

I'll see if I can test the change. I'd be happy to put up a different test case if needed with an actual component, however, the dev tools were greyed out for me component or not - and it seems this is officially closed anyway, so I guess should be fixed.
